src/main/java/org/broadinstitute/hellbender/engine/filters/ReadGroupReadFilter.java

```diff
@@ -8,7 +8,7 @@
 import java.io.Serializable;
 
 /**
- * Only use reads from the specified read group.
+ * Keep only reads from the specified read group.
```

Comment:
How does this filter behave if there are no read group tags on the reads? Will it keep all of them?